Frontend/android-native/app/src/main/assets/juspay/sdk_config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "logsConfig": {
+    "logsUrlKey": "https://logs.juspay.in/logs",
+    "logsUrlKeySandbox": "https://beta.logs.juspay.in/logs",
+    "pusherTimerKey": "10.0",
+    "encryptionLevelKey": "encryption",
+    "shouldPush": true,
+    "publicKeySandbox": {
+      "kty": "RSA",
+      "kid": "sandbox-log-24-03-2021",
+      "e": "AQAB",
+      "n": "v9SaTMmbyyPiFEv8NbqQo-y7nmcKLneS8jPkL9Xby04B9xI3wXkezAEjE9dNCd_vs8QEYVJTDIxOsxPTBHc5sm9WzswZMrs7qkr4FAdVpO-UOiiOTBbv8r2GdIR5Do8BFsAAeil1XsiXz15_r2VzawZujjnj6poUbpvfZ3wI2QV4ArVmpoSkGPBKEnpKeILXoz4xZrPCwjaLrAy8qeIWwbm13f0LxSJNHSpk3E0zdf4ZtgzWClqgnJEc4Rq53XNB6h3CaFNBKti5Dcoe3eUABafniOJC4K3gwWwQJ3kQUitpgMy0MnsfGjt2UgqLJTAx8ZZJVrNyFnjYq0QsZibsFw"
```

Comment:
Is it authorized or mandatory to put publicKey in repo? 

@kavya-shree-s @Vignesh-772